lib/tasks/user_limit.rake

```
@@ -15,15 +15,11 @@ namespace :user_limit do
     end
   end
 
-  def find_user_by_login(login)
-    User.find_by!("lower(login) = '#{login.downcase}'")
```

This rake task should raise the error..i'll fix this.